packages/contracts/test/test-utils/voting.ts

```diff
@@ -8,6 +9,27 @@ export enum VoteOption {
   No,
 }
 
+export enum VoteMode {
+  Default,
+  EarlyExecution,
+  VoteReplacement,
+}
+
+export enum Op {
```

Comment:
How did this end up here ? it should be in the common file.

Reply:
This was copied two times in the `addresslist-voting.ts` and `token-voting.ts` so I put it here.

Reply:
It's related to `auth` and permission manager, so it should be in `common.ts` or whatever.

Reply:
Fixed in fd397bb340f4befcb7a6d14f0832684e048eef7a